src/plugins/simulator/components/simulator-tile.tsx

```diff
@@ -0,0 +1,61 @@
+import { VariableType } from "@concord-consortium/diagram-view";
+import { observer } from "mobx-react";
+import React, { useEffect, useState } from "react";
+
+import { SimulatorContentModelType } from "../model/simulator-content";
+import { ITileProps } from "../../../components/tiles/tile-component";
+import { BasicEditableTileTitle } from "../../../components/tiles/basic-editable-tile-title";
+
+import "./simulator-tile.scss";
+
+export const SimulatorToolComponent: React.FC<ITileProps> = observer((props) => {
+  // Note: capturing the content here and using it in handleChange() below may run the risk
+  // of encountering a stale closure issue depending on the order in which content changes,
+  // component renders, and calls to handleChange() occur. See the PR discussion at
+  // (https://github.com/concord-consortium/collaborative-learning/pull/1222/files#r824873678
+  // and following comments) for details. We should be on the lookout for such issues.
+  const { model, readOnly, scale } = props;
+  const content = model.content as SimulatorContentModelType;
+
+  const [_steps, setSteps] = useState(0);
+  useEffect(() => {
+    const id = setInterval(() => {
+      content?.step();
+      setSteps(v => v + 1);
+    }, content.simulationData.delay);
+    return () => clearInterval(id);
+  }, [content]);
```

Comment:
This will be a good test of the "capturing the content" discussion above. I'm pretty sure MST will complain if content is removed from the tree and then step is called on it. So if we see those complaints then we'll have a case to explore about whether we need to add `useRef` here.